src/Server/HTTP/HTMLForm.cpp

```diff
@@ -183,8 +183,8 @@ void HTMLForm::readMultipart(ReadBuffer & in_, PartHandler & handler)
     size_t fields = 0;
     MultipartReadBuffer in(in_, boundary);
 
-    /// Assume there is at least one part
-    in.skipToNextBoundary();
+    if (!in.skipToNextBoundary())
+        throw Poco::Net::HTMLFormException("No boundary line found");
```

Comment:
Not sure about this, maybe it's better to just return without throwing an exception?

Reply:
Why not? If this function called at least one boundary should exists AFAICS

Reply:
I don't have strong opinion on this and I could replace it with simple return, but:
- exception provides more info (i.e. in case of some broken HTTP request)
- and plus if the client already goes away it will not see any error anyway

Reply:
I see now, lack of a boundary means an error, that's not a normal case, so I think it's better to throw an exception.